site/hxapi/transfers/av/index.md

```diff
@@ -0,0 +1,503 @@
+| Initials    | String    | [A-Z] 3 chars    | N          | The initials of the Operator/Agent.                                                                                   |
+| key         | String    | [A-Z]            | Y          | This will be assigned to you by your Account Manager during set up.                                                   |
+| token       | String    | [0-9] 9 chars    | Y          | Please see [user token endpoint](/hxapi/usertoken) for details of how to generate a token.                            |
+| PickUp      | String    | [A-Z0-9]         | Y          | The IATA, TTI or Postcode (i.e `PickUp=AGP`).                                                                         |
+| PickUpType  | String    | [A-Z0-9]         | Y          | The type can either be `IATA`, `TTI` or `PC` (Postcode) (i.e `PickUpType=IATA`).                                      |
+| DropOff     | String    | [A-Z0-9]         | Y          | The IATA, TTI or Postcode (i.e `DropOff=85101062`).                                                                   |
+| DropOffType | String    | [A-Z0-9]         | Y          | The type can either be `IATA`, `TTI` or `PC` (Postcode) (i.e `DropOffType=TTI`).                                      |
+| FromDate    | Date      | YYYY-MM-DD       | Y          | Flight Arrival Date (`PickUpType` = `IATA`), Flight Departure Date (`DropOffType` = `IATA`), Preferred Date.          |
+| FromTime    | Time      | HHMM             | Y          | Flight Arrival Time (`PickUpType` = `IATA`), Flight Departure Time (`DropOffType` = `IATA`), Preferred Time.          |
+| ReturnDate  | Date      | YYYY-MM-DD       | N          | Flight Arrival Date (`DropOffType` = `IATA`), Flight Departure Date (`PickUpType` = `IATA`), Preferred Date.          |
+| ReturnTime  | Time      | HHMM             | N          | Flight Arrival Time (`DropOffType` = `IATA`), Flight Departure Time (`PickUpType` = `IATA`), Preferred Time.          |
+| Adults      | Integer   | [0-9]            | Y          | Number of Adults requiring a transfer.                                                                                |
+| Children    | Integer   | [0-9]            | N          | Number of Children requiring a transfer.                                                                              |
+| Infants     | Integer   | [0-9]            | N          | Number of Infants requiring a transfer.                                                                               |
+| OutFlight   | String    | [A-Z0-9]         | Y*         | A flight number is required if the journey involves being picked up or dropped off at an Airport.                     |
+| ReturnFlight| String    | [A-Z0-9]         | N*         | A flight number is required if the return journey involves being picked up or dropped off at an Airport.              |
+
+#### Dates and Times
```

Comment:
Is it worth even more specifically calling out that people do not need to make adjustments themselves? I know thats something thats caused some issues in the past.